### PR TITLE
update zh-TW.json

### PR DESCRIPTION
**Pull request by BrokenPen**
improve zh-TW.json

translate English into Chinese(Taiwan),  reference using online dictionary: 
Taiwan， National Academy for Educational Research online dictionary ： http://terms.naer.edu.tw/

Also, use zh-CN.json as a compare the different of zh-CN and zh-TW words usage in the different country

### CODE REVIEW

File: source/locales/zh-TW.json [hunk 1]
```diff
@@ -214,24 +214,24 @@
...
-  "English": "英文",
-  "Spanish": "西班牙文",
-  "French": "法文",
-  "Italian": "義大利文",
-  "Dutch": "荷蘭文",
-  "Polish": "波蘭文",
-  "Portuguese": "葡萄牙文",
-  "Russian": "俄文",
-  "Swedish": "瑞典文",
+  "Catalan": "Catalan",
```
Comment (strugee) [inline source/locales/zh-TW.json]: Why are these being translated back to the English version?

File: source/locales/zh-TW.json [hunk 2]
```diff
@@ -162,50 +162,50 @@
...
   "server-os-notes": "--",
-  "File Encryption": "File encryption",
+  "File Encryption": "檔案加密",
   "disk-encryption-notes": "<p>2014-05-28: TrueCrypt currently has security issues and is not recommended.<p><p>If you’re running GNU/Linux, <strong>dm-crypt</strong> with <strong>LUKS</strong> is the recommended encryption option.</p><p>Gentoo GNU/Linux <a href='https://wiki.gentoo.org/wiki/DM-Crypt_LUKS'>maintains a guide</a> for <strong>dm-crypt</strong> with <strong>LUKS</strong>.</p><p>Arch GNU/Linux <a href='https://wiki.archlinux.org/index.php/Dm-crypt_with_LUKS'>maintains a guide</a> to <strong>dm-crypt</strong> with <strong>LUKS</strong>. Also useable for Parabola GNU/Linux.</p>",
-  "Mail Server": "Mail server",
-  "Groupware Server": "Groupware server",
-  "Mail Transfer Agent": "Mail transfer agent (MTA)",
+  "Mail Server": "郵件伺服器",
+  "Groupware Server": "群組軟體伺服器",
+  "Mail Transfer Agent": "郵件轉送代理者 (MTA)",
```
Comment (strugee) [inline source/locales/zh-TW.json]: Does this abbreviation make sense anymore given the translation?

Comment (BrokenPen) [inline source/locales/zh-TW.json]: You right, maybe just delete the abbreviation will be more concise.

**Comment by BrokenPen** [comment]
Of course, this is a native Chinese translate, but I am not a native Taiwanese, In fact, I am Hongkonger, I do understand the different computer words using in China Mainland, Taiwan, HongKong. checking the online dictionary to check whether is a zh-tw translation style.

File: source/locales/zh-TW.json (hunk 1 above)
Comment (BrokenPen) [inline source/locales/zh-TW.json]: oh, my mistake, seem other language translate do use the translation of the language name. (let me update the zh-CN.json  too)

**Comment by strugee** [comment]
Great, thanks @BrokenPen. Just wanted to make sure you had at least _some_ familiarity with the language :)

Would you mind if I cleaned up your commit messages a bit? This will involve force-pushing to your branch.

**Comment by BrokenPen** [comment]
＠strugee Do as you want, I not really understand how the git work. I need you help for that

**Comment by strugee** [comment]
Ugh, sorry! I screwed this up when I cleaned up the commits and force-pushed.

I went ahead and manually landed your changes in 8077436ca5a8ce6fa3729f4bcc0c98b2991e0abc. Thanks very much!

**Pull request closed without merging**